agir/groups/actions/notifications.py

```diff
@@ -94,9 +95,10 @@ def new_message_notifications(message):
 
     # Private message: notify only required memberships
     if message.message_type == SupportGroupMessage.MESSAGE_TYPE_ORGANIZATION:
-        recipients = message.supportgroup.memberships.filter(
+        memberships = message.supportgroup.memberships.filter(
             membership_type__gte=message.required_membership_type
         )
+        recipients = Person.objects.filter(memberships__in=memberships)
```

Comment:
Ça devrait être peut-être un peu plus rapide de chercher directement par id :
```suggestion
        recipients = Person.objects.filter(id__in=memberships.values_list("person_id", flat=True))
```